### DISCUSSION TITLE
How to Fix Compilation Errors : TS2344

### DISCUSSION THREAD

**Discussion by daeinlee** [vfsfitvnm/frida-il2cpp-bridge#148] [Q&A]
Hello. Thanks for making such a great tool.
I'm getting a compile error because of what I'm doing wrong. can i get any help

`F:\fridaagent2\node_modules\.bin>frida-compile index.ts -o out.js -c
[TypeScript error: F:/fridaagent2/node_modules/frida-il2cpp-bridge/dist/il2cpp/base.d.ts(34,41): Error TS2344: Type 'string' does not satisfy the constraint 'never'.
  Type 'string' is not assignable to type 'never'.] {
  fileName: 'F:/fridaagent2/node_modules/frida-il2cpp-bridge/dist/il2cpp/base.d.ts',
  line: 34,
  column: 41,
  inputs: Set(143) {
    'F:\\fridaagent2\\node_modules\\.bin\\index.ts',
    'F:/fridaagent2/_agent.js',`

![compileError](https://user-images.githubusercontent.com/34927021/158128831-ab29e403-aad7-4c62-854b-5d83f9dda275.JPG)

**Comment by vfsfitvnm** [comment]
Hi, this is a TS issue. Please delete `out.js` and retry, also make sure to follow this guide https://github.com/vfsfitvnm/frida-il2cpp-bridge/wiki/Installation#from-scratch

**Reply by daeinlee** [reply]
Thank you so much for the reply. It's difficult because I don't have a lot of knowledge about TypeScript. 
We will look at your reply and try again.
